src/api/server-routes.ts

```diff
@@ -402,6 +403,24 @@ export function permissionRoutes(): express.Router {
   return router;
 }
 
+export function environmentVariableRoutes(): express.Router {
+  const router = express.Router();
+
+  router.post('/add', (req: express.Request, res: express.Response) => {
+    insertEnvironmentVariable(req.body)
+      .then(() => res.status(200).json({ data: true }))
+      .catch(() => res.status(200).json({ data: false }));
+  });
+
+  router.get('/remove/:id', (req: express.Request, res: express.Response) => {
+    removeEnvironmentVariable(req.params.id)
```

Comment:
this route is totally unprotected, anyone can delete env variables with accessing this route and break the builds eventually.